sdcm/ec2_client.py

```diff
@@ -353,3 +356,21 @@ def get_instance_by_private_ip(self, private_ip):
             raise GetInstanceByPrivateIpError("Cannot find instance by private ip: %s" % private_ip)
 
         return self.get_instance(instances['Reservations'][0]['Instances'][0]['InstanceId'])
+
+    def add_placement_group_name_param(self, boto3_params):
+        config = TestConfig()
+        sct_params = SCTConfiguration()
+        placement_group_name = f"{sct_params.get('user_prefix')}-placement_group-{config.test_id()[:8]}"
+        placement_group_tags = {"TestId": config.test_id(), "Name": placement_group_name}
+        list_placement_groups = list_placement_groups_aws(
+            placement_group_tags, self.region_name)
+        if sct_params.get("use_placement_group"):
```

Comment:
why not to move this at top of function:
``` python
if not sct_params.get("use_placement_group"):
    return
```
Then there will be no nested if and no need several calls like TestConfig() and SCTConfiguration()

WDYT?

Reply:
updated